File: 프로그래머스/lv0/120842. 2차원으로 만들기/2차원으로 만들기.py
def solution(num_list, n):
    length = len(num_list) // n
    answer = [[0] * n for _ in range(length)] 
    idx = 0 

    for i in range(length):
        for j in range(n):
            answer[i][j] = num_list[idx]
            idx += 1

    return answer

File: 프로그래머스/lv0/120842. 2차원으로 만들기/2차원으로 만들기.java
class Solution {
    public int[][] solution(int[] num_list, int n) {
        
        int len = num_list.length / n;
        int idx = 0;
        int[][] answer = new int[len][n];
        for(int i =0; i<len; i++){
            for(int j=0; j<n; j++){
                answer[i][j] = num_list[idx++];
            }
        }
        return answer;
    }
}